$NetBSD: patch-arch_arm_dts_sun50i-h5-nanopi-neo-plus2.dts,v 1.3 2018/05/25 20:49:52 jakllsch Exp $

Enable use of the NanoPi Neo Plus2's onboard Ethernet.

Templated from the orangepi-pc2.dts.

--- arch/arm/dts/sun50i-h5-nanopi-neo-plus2.dts.orig	2018-05-07 15:32:36.000000000 +0000
+++ arch/arm/dts/sun50i-h5-nanopi-neo-plus2.dts
@@ -104,3 +104,18 @@
 &usbphy {
 	status = "okay";
 };
+
+&emac {
+	pinctrl-names = "default";
+	pinctrl-0 = <&emac_rgmii_pins>;
+	phy-mode = "rgmii";
+	phy-handle = <&ext_rgmii_phy>;
+	status = "okay";
+};
+
+&external_mdio {
+	ext_rgmii_phy: ethernet-phy@7 {
+		compatible = "ethernet-phy-ieee802.3-c22";
+		reg = <7>;
+	};
+};
